docs/en/install.md

```diff
@@ -16,8 +16,8 @@ MMOCR has different version requirements on MMCV and MMDetection at each release
 
 | MMOCR        | MMCV                   | MMDetection               |
 | ------------ | ---------------------- | ------------------------- |
-| master       | 1.3.8 <= mmcv <= 1.6.0 | 2.14.0 <= mmdet <= 3.0.0  |
-| 0.5.0        | 1.3.8 <= mmcv <= 1.5.0 | 2.14.0 <= mmdet <= 3.0.0  |
+|   main       | 1.3.8 <= mmcv <= 1.6.0 | 2.21.0 <= mmdet <= 3.0.0  |
+| 0.5.0        | 1.3.8 <= mmcv <= 1.5.0 | 2.21.0 <= mmdet <= 3.0.0  |
```

Comment:
#944 was merged after the release of 0.5.0. 0.5.0 is still compatible with mmdet>=2.14.0

Reply:
And there are extra spaces before "main"